docs/REST_API.md

```diff
@@ -230,6 +233,12 @@ GET /config
       "engine": "ScanPy version 1.33",
       "dataset": "/home/joe/mouse/blorth.csv"
     },
+    "parameters": {
+      "verbose": false,
+      "log-level": "info",
+      "minimumThreshold", 99
```

Comment:
why is this minimum and not maximum?

Reply:
just another random example...could be anything.